cpp/include/cudf/strings/detail/convert/fixed_point.cuh

```diff
@@ -121,7 +121,7 @@ __device__ thrust::optional<int32_t> parse_exponent(char const* iter, char const
  * @brief Converts the string in the range [iter, iter_end) into a decimal.
  *
  * @tparam DecimalType The decimal type to be returned
- * @param iter The beginning of the string. Unless iter >= iter_end, iter is dereferenced
+ * @param iter The beginning of the string Unless iter >= iter_end, iter is dereferenced
```

Comment:
Probably not a good idea to just globally remove the period. The number of files to review is very large now and most of these are just not worth doing IMO since these do not cause doxygen warnings. My review comments was mainly to try to honor the no-period for non-sentence in any _new_ comments rather than make global changes to all existing comments.
```suggestion
 * @param iter The beginning of the string unless iter >= iter_end, iter is dereferenced
```